components/about/contact-person.tsx

```diff
@@ -48,35 +54,47 @@ export const ContactPerson: React.FC<ContactPersonProps> = ({
             />
           </AspectRatio>
           <Grid
-            sx={{
-              gridTemplateRows: [
-                'auto',
-                '26px 21px 24px 24px 24px 24px',
-                '26px 21px 24px 24px 24px 24px',
-              ],
+            templateRows={{
+              base: 'auto',
+              md: '3rem 2rem 1.5rem 2rem 2rem 2rem',
             }}
           >
-            <Text sx={{ fontSize: 2, fontWeight: '600', mt: '20px' }}>
+            <Text
+              sx={{
+                fontSize: 'base',
+                fontWeight: 'semibold',
+                mt: '20px',
```

Comment:
Do we need more spacing options in the theme file so we can avoid using `px` here?

Reply:
I was going to do something similar to the thing with fontsize - adaptible spacing depending on page size using this: https://utopia.fyi/space/calculator/ tool. But i didnt get around to it yet :)